### PR TITLE
chore: added logo

### PR DESCRIPTION
**Pull request by Magouo**

### CODE REVIEW

File: src/components/Navbar.tsx [hunk 1]
```diff
@@ -12,7 +12,17 @@ import {
   useDisclosure,
 } from "@chakra-ui/react";
 import { HamburgerIcon, CloseIcon } from "@chakra-ui/icons";
+import Image from "next/image";
```
Comment (Byczax) [inline src/components/Navbar.tsx:15]: change `import Image` into `import NextImage`

File: src/components/Footer.tsx [hunk 2]
```diff
@@ -78,8 +61,15 @@ export const Footer = () => {
         justify={{ base: "center", md: "space-between" }}
         align={{ base: "center", md: "center" }}
       >
-        <Logo />
-        <Text>Komisja ds. Informatyzacji</Text>
+        <Image src={logoPwr as string} alt="logo-pwr" object-fit="contain" />
+        <Image
+          src={logoSSPwr as string}
+          alt="logo-sspwr"
+          object-fit="contain"
```
Comment (Byczax) [inline src/components/Footer.tsx:68]: is there need for this parameter?
Right now every logo is in original size, in the next task there will be need to scale them.

**Review by Byczax — CHANGES_REQUESTED** [review]

**Review by Rei-x — CHANGES_REQUESTED** [review]
popraw alty

**Review by Byczax — APPROVED** [review]
ready to merge! 🎉

**Review by Rei-x — APPROVED** [review]
g

**Comment by Rei-x** [comment]
closes #7

**Comment by Rei-x** [comment]
fixes #7

**Pull request merged**